docs/user/02-operation-guides/troubleshooting/03-50-recovering-from-unintentional-istio-removal.md

```diff
@@ -1,53 +1,80 @@
 ```
 kubectl get istio default -n kyma-system -o jsonpath='{.status.description}'
+
 There are Istio resources that block deletion. Please take a look at kyma-system/istio-controller-manager logs to see more information about the warning
 ```
+#### **Kyma Dashboard**
+
+![Istio CR in the Warning state](../../../assets/istio-cr-warning-state.svg)
+<!-- tabs:end -->
 
 ## Cause
 
-- Istio wasn't completely removed because the user's CRs still exist.
+Istio wasn't completely removed because the user's CRs still exist.
 
-For example, the issue occurs when you delete Istio, but there are still `VirtualService` resources that either belong to the user or were installed by another Kyma component or module. In such cases, the hooked finalizer pauses the deletion of Istio until you remove all the related resources. This [blocking deletion strategy](https://github.com/kyma-project/community/issues/765) is intentionally designed and is enabled by default for Kyma Istio Operator.
+For example, the issue occurs when you delete Istio, but there are still Virtual Service resources that either belong to the user or were installed by another Kyma component or module. In such cases, the hooked finalizer pauses the deletion of Istio until you remove all the related resources. This [blocking deletion strategy](https://github.com/kyma-project/community/issues/765) is intentionally designed and is enabled by default for Kyma Istio Operator.
 
 
 ## Remedy
 
- 1. Edit the Istio CR and remove the finalizer.
-```
-  kubectl edit istio -n kyma-system default
-  ```
-  ```diff
-  apiVersion: operator.kyma-project.io/v1alpha1
-  kind: Istio
-  metadata:
-  < finalizers:
-  < - istios.operator.kyma-project.io/istio-installation
-    generation: 2
-    name: default
-    namespace: kyma-system
-  spec:
-    ...
-  status:
-    description: 'There are Istio resources that block deletion. Please take a look at kyma-system/istio-controller-manager logs to see more information about the warning'
-    state: Warning
-```
- 2. When the finalizer is removed, the Istio CR is deleted. Other resources, such as the `istiod` deployment, remain on the cluster.
+1. Edit the Istio CR and remove the finalizer.
+
+    <!-- tabs:start -->
+    #### **kubectl**
+
+    Run the following command to edit the Istio CR:
+    ```
+    kubectl edit istio -n kyma-system default
+    ```
+    Remove the indicated lines:
+    ```diff
+    apiVersion: operator.kyma-project.io/v1alpha1
+    kind: Istio
+    metadata:
+    < finalizers:
+    < - istios.operator.kyma-project.io/istio-installation
+      generation: 3
+      labels:
+        ...
+      name: default
+      namespace: kyma-system
+        ...
+      status:
+        description: 'There are Istio resources that block deletion. Please take a look at kyma-system/istio-controller-manager logs to see more information about the warning'
+        state: Warning
+    ```
+    
+    #### **Kyma Dashboard**
+
+   1. Go to the kyma-system Namespace. 
```

Comment:
I'm not sure - should "kyma-system" be formatted as code font (because it's a technical name?) or as bold font (because it appears in the UI?). Or should it remain in plain text?

Reply:
It shouldn't be written in plain text, for sure. I changed it to code font, but I think using bold would also be fine